### ISSUE TITLE
Czech description?

### ISSUE THREAD

**Issue by VladimirAlexiev** [EKGF/dprod#96]
https://ekgf.github.io/dprod/#dataservice-endpointurl
> Kořenové umístění nebo hlavní přístupový bod služby (IRI přístupné přes Web).

Uh? How did it pick up the `cz` value?

**Comment by rivettp** [comment]
I think we're pulling those from DCAT which I know has many translations, but we should have our own local definition for eht shape.
Same issue with the label for that and endPointDescription

**Comment by jgeluk** [comment]
Yep, I I know what caused that. Me. I need to fix that Python code, filtering out labels with other language labels.

**Comment by rivettp** [comment]
Fixed in the official OMG specification before adoption.
